FSJ_django20_project/FSJ/templates/FSJ/coord_home.html

```diff
@@ -0,0 +1,20 @@
+{% extends "FSJ/base_with_banner.html" %}
```

Comment:
This template isn't used by the code from what I can see. The coordinator home page is home.html not coord_home.html

Reply:
I've deleted this coord_home.html, it was from an early test